src/check_missing_alleles.py

```diff
@@ -0,0 +1,439 @@
+import csv
+import os
+from subprocess import Popen, PIPE
+
+
+def get_IMGT_data():
```

Comment:
Why not get these files using `curl` in the `Makefile` and pass their paths as arguments to this script?

Reply:
(and downloaded into the `build/` directory instead of the working directory)